pyproject.toml

```diff
@@ -37,7 +37,7 @@ version_file = "src/posit/_version.py"
 
 [tool.ruff]
 extend-exclude = ["integration/resources"]
-line-length = 79
+line-length = 99
```

Comment:
How did you decide on 99? I need to update neovim with a config like this so that I get a visual when my lines are too long (mostly for doc writing since markdown handles wrapping automatically) but I can't decide on a good global default. The standard terminal width 80 feels too small and I think that would be annoying

Reply:
Does NeoVim support line wraps?

Reply:
Probably but I don't actually want the automatic wrapping in most cases because it tends to cause such large diffs when I edit files in repos that don't have any formatting enforcement. I prefer the visual so I can see when I'm editing a line that exceeds the length and I can just fix it while I'm in there for lines that I'm already changing.